### DISCUSSION TITLE
Docker-compose部署Claude 3 API配置

### DISCUSSION THREAD

**Discussion by lihetongxue** [lobehub/lobehub#1758] [Self Hosting]
使用Docker-compose部署网页，以下是我的docker-compose.yml文件配置：
<img width="958" alt="屏幕截图 2024-03-27 115959" src="https://github.com/lobehub/lobe-chat/assets/97779346/7d7be064-55da-4aa2-b959-3969395de3b2">
在使用的时候返回了403错误，信息是“Request not allowed”
我的配置有问题吗，还是我的api的问题，求助！

**Comment by BrandonStudio** [comment]
See #1649

**Reply by lihetongxue** [reply]
我看了，意思是我的API的问题对吧
我并没有找到原因

**Reply by BrandonStudio** [reply]
意思是你在Claude不支持的地方调用了API

要么换IP，要么套Cloudflare

**Reply by lihetongxue** [reply]
噢噢，我明白了，想请教以下怎么套Cloudflare

**Reply by BrandonStudio** [reply]
https://developers.cloudflare.com/ai-gateway/get-started/creating-gateway/
